### PR TITLE
add asset constants for localized strings, images and colors

### PR DESCRIPTION
**Pull request by johannesauer**

### CODE REVIEW

File: macos/asset/Color.swift [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+import AppKit
+
+let AssetAppColor = NSColor(named: "AppColor")!
+let AssetAppUnemphasizedColor = NSColor(named: "AppColorUnemphasized")!
+let AssetTextFieldBackgroundColor = NSColor(named: "TextFieldBackgroundColor")!
```
Comment (NicoSa) [inline macos/asset/Color.swift]: I would prefer if for example `AssetAppColor` would be called like this instead `Color.app` or `AssetWatermarkColor` with `Color.watermark`

File: macos/asset/String.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
+
+import Foundation
+
+let AssetAboutAppString = NSLocalizedString("AboutApp", comment: "")
+let AssetAppNameString = NSLocalizedString("AppName", comment: "")
```
Comment (NicoSa) [inline macos/asset/String.swift]: `Localization.aboutApp`

File: macos/asset/Image.swift [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+import AppKit
+
+let AssetAddImage = NSImage(templateNamed: "Add")!
+let AssetBulbImage = NSImage(templateNamed: "Bulb")!
+let AssetFileImage = NSImage(templateNamed: "File")!
+let AssetFileLargeImage = NSImage(templateNamed: "FileLarge")!
+let AssetFilePlaceholderImage = NSImage(templateNamed: "FilePlaceholder")!
+let AssetLockImage = NSImage(templateNamed: "Lock")!
```
Comment (NicoSa) [inline macos/asset/Image.swift]: `Image.lock`

File: macos/asset/Color.swift (hunk 1 above)
Comment (johannesauer) [reply macos/asset/Color.swift]: done

File: macos/asset/String.swift (hunk 2 above)
Comment (johannesauer) [reply macos/asset/String.swift]: done

File: macos/asset/Image.swift (hunk 3 above)
Comment (johannesauer) [reply macos/asset/Image.swift]: done

**Pull request merged**